pyphi/subsystem.py

```diff
@@ -383,14 +385,14 @@ def _single_node_effect_repertoire(
         self,
         condition: FrozenMap[int, int],
         purview_node_index: int,
-        use_backward_tpm: bool
+        direction: Direction
     ):
         # pylint: disable=missing-docstring
         purview_node = self._index2node[purview_node_index]
         # Condition on the state of the purview inputs that are in the mechanism
-        if use_backward_tpm:
+        if direction == Direction.CAUSE:
             tpm = purview_node.backward_tpm.condition_tpm(condition)
-        else:
+        elif direction == Direction.EFFECT:
```

Comment:
Add an `else` block and call [validate.direction](https://github.com/wmayner/pyphi/blob/6b83cbdbbcdca75289415fe096adbac5f2ec7a4d/pyphi/validate.py#L21) with `allow_bi=False` to raise the appropriate error.